website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md

```diff
@@ -13,15 +13,15 @@ dbt Cloud provides a helpful interface around dbt's [source data freshness](usin
 
 First, make sure to configure your sources to [snapshot freshness information](using-sources#snapshotting-source-data-freshness).
 
-Then, to enable source freshness snapshots in dbt Cloud, add a `dbt source snapshot-freshness` step to one of your jobs, or create a new job to snapshot source freshness.
+Then, to enable source freshness snapshots in dbt Cloud, add a `dbt source freshness` step to one of your jobs, or create a new job to snapshot source freshness.
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/49a03cc-Screen_Shot_2019-03-06_at_10.24.15_AM.png" title="Adding a step to snapshot source freshness"/>
 
-You can add `dbt source snapshot-freshness` anywhere in your list of run steps, but note that if your source data is out of date, this step will "fail', and subsequent steps will not run. dbt Cloud will trigger email notifications (if configured) based on the end state of this step.
+You can add `dbt source freshness` anywhere in your list of run steps, but note that if your source data is out of date, this step will "fail', and subsequent steps will not run. dbt Cloud will trigger email notifications (if configured) based on the end state of this step.
 
-If you *do not* want your models to run if your source data is out of date, then it could be a good idea to run `dbt source snapshot-freshness` as the first step in your job. Otherwise, we recommend adding `dbt source snapshot-freshness` as the last step in the job, or creating a separate job just for this task.
+If you *do not* want your models to run if your source data is out of date, then it could be a good idea to run `dbt source freshness` as the first step in your job. Otherwise, we recommend adding `dbt source freshness` as the last step in the job, or creating a separate job just for this task.
```

Comment:
Would it be helpful to have some information here about how this interaction plays out in the new 'dbt build' command?

Reply:
good call! added in https://github.com/dbt-labs/docs.getdbt.com/pull/831/commits/fb40346c5952eb505efeca154d5cd6cb2e108a9d